usr/sbin/p11sak/p11sak.c

```diff
@@ -3040,11 +3040,11 @@ static CK_RV delete_key(CK_SESSION_HANDLE session, p11sak_kt kt, char *rm_label,
 
     if (strlen(rm_label) > 0) {
         if (boolDestroyFlag) {
-            printf("Object with Label: %s found and destroyed \n", rm_label);
+            printf("Object with Label \"%s\" found and destroyed \n", rm_label);
         } else if (boolSkipFlag) {
-            fprintf(stderr, "Object with Label: %s not deleted\n", rm_label);
+            fprintf(stderr, "Object with Label \"%s\" not deleted\n", rm_label);
         } else if (rc == CKR_OK) {
```

Comment:
Above, we are talking about "destroying" a key, rather than "deleting" a key. Maybe it's better to also say "destroy" here? However, in general I would prefer the term "delete" at all places in p11sak.

Reply:
Well, the PKCS#11 function to delete a object is called `C_DestroyObject()`, there is nothing we can do about that :-) The function where we are in is called `delete_key()` and the p11sak command that it belongs to is named `remove-key`, so we have all 3 possible terms used, mission accomplished ;-)